### PR TITLE
Double Ice Cream, a new drink for bartenders to serve. Take 2

### PR DESCRIPTION
**Pull request by dge21**
## About the PR

Added a new non alchoholic drink, now in the delta V folders
## Why / Balance

its something I already used to mix, but people don't really pay attention to the "tastes like" messages, and this means the drink is more official looking and doesn't rely on me prompting them to pay attention to the taste like message
## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None probably
**Changelog**

:cl:
- add: added a sweet new drink for the bar to serve

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Drinks/drinks.yml [hunk 1]
```diff
@@ -133,3 +133,19 @@
...
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+        - ReagentId: DoubleIceCream
+          Quantity: 30
+  - type: Icon
+    sprite: Objects/Consumable/Drinks/doubleicecreamglass.rsi
```
Comment (TadJohnson00) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Drinks/drinks.yml]: Should read `DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi`

File: Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/meta.json
Comment (TadJohnson00) [inline Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/meta.json:1]: You do not need delays for static sprites. Please remove them

File: Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/Icon_empty.png
Comment (TadJohnson00) [inline Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/Icon_empty.png:1]: If you could remove the capitalisation on this file name that'd be sweet

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]
Please address changes, otherwise should be good for merge

File: Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/meta.json
Comment (dge21) [reply Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/meta.json:1]: how would I do that?

Comment (dge21) [reply Resources/Textures/DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi/meta.json:1]: wait I think I figured it out

File: Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml [hunk 2]
```diff
@@ -205,3 +205,19 @@
...
+- type: reagent
+  id: DoubleIceCream
+  name: reagent-name-double-ice-cream
+  parent: BaseSoda
+  desc: reagent-desc-double-ice-cream
+  physicalDesc: reagent-physical-desc-fizzy-and-creamy
+  flavor: doubleicecream
+  color: "#a06f42"
+  metamorphicSprite:
+    sprite: Objects/Consumable/Drinks/doubleicecreamglass.rsi
```
Comment (TadJohnson00) [inline Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml]: This should path to DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]
Should be only one final change until this PR passes tests.

File: Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml (hunk 2 above)
Comment (dge21) [reply Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml]: alright, should be done now

**Comment by TadJohnson00** [comment]
Hype

**Review by TadJohnson00 — APPROVED** [review]

**Pull request merged**